WeakAurasOptions/AceGUI-Widgets/AceGUIWidget-WeakAurasDisplayButton.lua

```diff
@@ -44,6 +44,35 @@ local function Show_Long_Tooltip(owner, description)
     GameTooltip:Show();
 end
 
+local function IsOverDisplay(display)
+    if not display then return nil end
+
+    local button = WeakAuras.GetDisplayButton(display)
+    return MouseIsOver(button.frame)
```

Comment:
So if I read that correctly, http://wowwiki.wikia.com/wiki/API_MouseIsOver supports a top and bottom offset.

I still think that it should behave such that hoovering over the top half of a buttom should insert above it, and hoovering over the bottom half below that aura. 

That should be implementable by using half a width as bottom and topoffset (no idea on the sign), in such a way that the button steals the a half of the next button.